packages/node/src/BacktraceClient.ts

```diff
@@ -1,15 +1,16 @@
 import {
     BacktraceAttributeProvider,
-    BacktraceConfiguration,
+    BacktraceConfiguration as CoreConfiguration,
     BacktraceCoreClient,
     BacktraceRequestHandler,
 } from '@backtrace/sdk-core';
 import { AGENT } from './agentDefinition';
+import { BacktraceConfiguration } from './BacktraceConfiguration';
 import { BacktraceClientBuilder } from './builder/BacktraceClientBuilder';
 
 export class BacktraceClient extends BacktraceCoreClient {
     constructor(
-        options: BacktraceConfiguration,
+        options: CoreConfiguration,
```

Comment:
Because there is a type mismatch between BacktraceConfiguration that someone in Node wants to configure and the definition the core library expects. 

tldr:

the core library expects specific type of attachments that follow the interface. The node library allows to define attachment in simple type like string and let the library convert it